back-end/controllers/UserController.js

```diff
@@ -18,6 +24,14 @@ const updateUserById = async (req, res) => {
   try {
     const { id } = req.params;
     const user = await updateByID(id, req.body);
+    if (req.body.url) {
+      if (user.role === STUDENT) {
+        await updateStudent(user.roleId, req.body);
```

Comment:
When we change photo student/teacher methods should receive only photo url not all user updating data

Reply:
fix it